webserver_config.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you remove commented code from this file and leave a comment as to why it's necessary?